src/components/async-resource/simple-async-resource.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import Spinner from '../spinner';
+import Alert from '../alert';
+
+import externalUrls from '../../externalUrls';
+import requestStates from '../../state/state-utils/request-states';
+
+const simpleAsyncResource = props => {
```

Comment:
React components must start with uppercase. This works because it's being exported as the module's default export and then imported into a variable starting with uppercase. This will cause linting errors.

It's nice to also export the named component as well, for testing it isolated from decorators like mapRouteParamsToProps.